src/Snap/Internal/Http/Server/Config.hs

```diff
@@ -130,6 +138,15 @@ instance Show ConfigLog where
     show (ConfigFileLog f) = "log to file " ++ show f
     show (ConfigIoLog _)   = "custom logging handler"
 
+------------------------------------------------------------------------------
+-- | This handler may be used (in conjunction with setErrorLogHandler) to write out error logs in a
+-- custom manner.
+type ErrorLogHandler = ByteString -> IO ByteString
```

Comment:
@gregorycollins So you want me to change the type to Builder -> IO Builder? Because otherwise the ErrorLogHandler would actually have to write the logs to the ConfigLog, rather than just producing logs in a custom format as it currently does.